control/mpc_lateral_controller/include/mpc_lateral_controller/mpc_lateral_controller.hpp

```diff
@@ -88,9 +91,16 @@ class MpcLateralController : public trajectory_follower::LateralControllerBase
   // Flag indicating whether to keep the steering control until it converges.
   bool m_keep_steer_control_until_converged;
 
+  // MPC colver checker.
```

Comment:
```suggestion
  // MPC solver checker.
```

Reply:
fixed in 5a43cdb4cc009c0be2591181f2ee99b84a6a1f3c